js/src/tab.js

```diff
@@ -287,15 +287,15 @@ EventHandler.on(window, Event.LOAD_DATA_API, () => {
     let selectedTabFound = false
 
     // iterate over each tab in the tablist, make sure they have correct tabindex/aria-selected
-    for (let j = 0; j < tabs.length; j++) {
-      if (tabs[j].getAttribute('aria-selected') === 'true' && !selectedTabFound) {
-        tabs[j].setAttribute('tabindex', '0')
+    tabs.forEach(tab => {
```

Comment:
You should use our helper function `makeArray` here.

Reply:
any pointers of how to do that would be welcome. as said, i've got very little idea of the idiosyncracies of our helper functions, admittedly...

Reply:
You can look at how it's already being used like here https://github.com/twbs/bootstrap/blob/master/js/src/carousel.js#L337

Reply:
tried to tighten that into `makeArray(SelectorEngine.find(Selector.DATA_TOGGLE, tablists[i])).forEach(tab => {` but then not obvious how after the loop i can revisit the first tab to set it unless i run some other SelectorEngine thing again, which seems wasteful/pointless?

Reply:
There's a reason we have this utility. Some browsers don't support `NodeList.forEach`

Reply:
but i am using makeArray a few lines earlier...not quite sure i follow what you're saying here?

Reply:
as in `tabs` is defined a few lines earlier already as `const tabs = makeArray(SelectorEngine.find(Selector.DATA_TOGGLE, tablists[i]))` ... just that i'm not currently chaining the `forEach` directly after it but storing it in the variable

Reply:
Ah, nvm then, it should be fine.